notebooks/measures/Search_Download_and_Plot_GeoTIFFs.ipynb

```diff
@@ -0,0 +1,503 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55d7625f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Set up an outputs folder to download the data to\n",
+    "path = str(os.getcwd() + '/outputs')\n",
+    "if not os.path.exists(path):\n",
+    "    os.mkdir(path)\n",
+    "\n",
+    "#Download the DEM granules \n",
+    "dem_files = earthaccess.download(results_dem, path)\n",
+    "\n",
+    "#Download the velocity granules\n",
+    "vel_files = earthaccess.download(results_vel, path)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db0008d8",
+   "metadata": {},
+   "source": [
+    "### **Check the files that have been downloaded**\n",
+    "We will list all the DEM and velocity files that were downloaded, as this is needed for the next steps. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0095777f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dir_list = os.listdir(path)\n",
+    "\n",
+    "print('Files in ', path)\n",
+    "\n",
+    "for x in dir_list:\n",
+    "    if x.endswith('.tif'):\n",
+    "        print(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5afa949b",
+   "metadata": {},
+   "source": [
+    "### **Select and read in the DEM file and velocity file**\n",
+    "\n",
+    "Based on the list of filenames above, we will select the GeoTIFFs that we wish to plot and input them into the cell below. We will use the 'browse' GeoTIFF of the DEM tile, as that provides the best continuous surface for visual display. For the velocity we will plot the velocity magnitude, which is denoted by 'vv' in the filename, and we will plot the velocity covering the time period 07 August to 18 August 2017. \n",
+    "\n",
+    "The DEM GeoTIFF covers a larger area than the velocity GeoTIFF, and we will be cropping the DEM GeoTIFF to the extent of the velocity GeoTIFF, so we will also set a filename for the cropped DEM GeoTIFF. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "36027699",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vel_fp = str(path + '/TSX_E68.80N_07Aug17_18Aug17_19-41-22_vv_v04.0.tif')\n",
+    "\n",
+    "dem_fp = str(path + '/tile_4_2_30m_browse_v02.0.tif')\n",
+    "\n",
+    "dem_crop = str(path + '/dem_crop_100.tif')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39178989",
+   "metadata": {},
+   "source": [
+    "### **Crop and resample DEM file based on velocity file extent and pixel size**\n",
+    "\n",
+    "We will use the `open` method from the `gdal` module to read the velocity GeoTIFF and get the extent and pixel size, and we will use the `SpatialReference` method from the `osr` module to get the projection information. We will then use this information to crop and downsample the DEM GeoTIFF using the `Warp` method from the `gdal` module."
```

Comment:
I wonder if we could split this up a bit more since the following code block is fairly long with a few separate steps. Since you have the code block split into three separate sections, maybe you could break this up into three separate markdown explanations + code block sections. You could also include comments within the code blocks to explain what each line is doing.

Reply:
Good idea, have made that change